tests/bluetooth/tester/src/gatt.c

```diff
@@ -1511,7 +1512,7 @@ static void read_long(uint8_t *data, uint16_t len)
 		   BTP_STATUS_FAILED);
 }
 
-static void read_multiple(uint8_t *data, uint16_t len)
+static void read_multiple(uint8_t *data, uint16_t len, bool variable_length)
```

Comment:
I think the code would be more readable if you just passed the opcode instead of a `bool` since you can't directly know what true/false means just by looking at the call to the function. So I'd propose to make the last parameter `uint16_t opcode`.

You can then also combine the two cases in the switch statement:
```
case GATT_READ_MULTIPLE:
case GATT_READ_MULTIPLE_VAR:
        read_multiple(data, len, opcode);
        return;
```

Reply:
Yes, it does make the callsite more readable.